ipa-core/src/protocol/hybrid/mod.rs

```diff
@@ -83,7 +86,14 @@ where
     // TODO shuffle input rows
     let shuffled_input_rows = padded_input_rows;
 
-    let _prf_input_rows = compute_prf_for_inputs(ctx.clone(), &shuffled_input_rows).await?;
+    let prf_input_rows_stream = compute_prf_for_inputs(ctx.clone(), &shuffled_input_rows).await?;
+
+    let _sharded_prf_rows = reshard_iter(
+        ctx.narrow(&Step::ReshardByPrf),
+        prf_input_rows_stream,
+        |ctx, _, report| report.shard_picker(ctx.shard_count()),
```

Comment:
is this what you want me to work on?

Reply:
yes, though most the work is changing the function above `compute_prf_for_inputs` to return a TryStream instead of a Vec. at that point, we should be able to replace `reshard_iter` with `reshard_try_stream` here.

Reply:
I looked into that and decided that changing `compute_prf_for_inputs` is better. Returning a stream that borrows from local variables is hard in Rust, so doing in-place resharding is better. Besides, this function is already async and returning a stream from already async function is a bit weird

Reply:
#1425 should address that